.github/workflows/build-wireguard-go-linux.yml

```diff
@@ -31,10 +35,19 @@ jobs:
         run: |
           ./wireguard/build-wireguard-go.sh
 
+      - name: Set artifact name
+        id: set_artifact_name
+        run: |
+          if [ -z "${{ inputs.artifact_name }}" ]; then
+            echo "artifact_name=wireguard-go_ubuntu-22.04_x86_64" >> $GITHUB_OUTPUT
+          else
+            echo "artifact_name=${{ inputs.artifact_name }}" >> $GITHUB_OUTPUT
```

Comment:
Heads up, if Im not wrong, using outputs requires you setup the job's top level `outputs` property for it to work

Reply:
ps: you can [inspect logs](https://github.com/nymtech/nym-vpn-client/actions/runs/15829426386/job/44618046959?pr=2999#step:7:21) from "upload artifact"  and see that the output is properly picked up. Seems to work as intended.